### ISSUE TITLE
Table: Table action row headers now output the scope attribute with no assignment breaking cypress a11y tests

### ISSUE THREAD

**Issue by jcaianirh** [patternfly/patternfly-react#5373]
**Describe the issue. What is the expected and unexpected behavior?**
PF Table action row headers such as this example:
https://www.patternfly.org/v4/components/table#actions-and-first-cell-in-body-rows-as-th
output a scope attribute that has no assignment causing this a11y test to fail:
https://dequeuniversity.com/rules/axe/3.4/scope-attr-valid?application=axeAPI
This is breaking openshift cypress a11y tests and causing us not to be able to upgrade.
**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

https://www.patternfly.org/v4/components/table#actions-and-first-cell-in-body-rows-as-th
inspect the action row and see there is an empty scope
**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
bug
**What is your product and what release version are you targeting?**
openshift 4.7

**Comment by jcaianirh** [comment]
This bug was introduced in 2020.15, you can see that in 2020.14 the extra scope attribute does not exist here:
https://www.patternfly.org/2020.14/components/table/#simple-actions

**Comment by tlabaj** [comment]
closed by #5253

**Comment by jessiehuff** [comment]
It looks like this issue was resolved in [this PR](https://github.com/patternfly/patternfly-react/pull/5253). 🙂
